FlowCrypt/Model/Token.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Token.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 25.11.2019.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+final class Token: Object {
```

Comment:
If this is access token, then we should probably call it `EmailAccessToken`.

Maybe easier to keep access token as a column in User table? Making a separate table may be an overkill.

Reply:
It's easier to keep them separate. And I don't whant to have whole user in places where I just need a token. For example some "ServerManager" don't need to know about User itself it just need a token
Renamed with `EmailAccessToken`

Reply:
Ok, got it. Lets do it your way then, and keep them separate.